jwql/website/apps/jwql/templates/view_image.html

```diff
@@ -47,6 +47,7 @@ <h3>{{ file_root }}</h3>
     		    <span class="image_preview">
     		    	<img id="image_viewer"
                          src='{{ static("") }}preview_images/{{ file_root[:7] }}/{{ file_root }}_cal_integ0.jpg'
+                         onerror="imageError(this, true);"
```

Comment:
Yup, pretty much. If there is an error with this image (like "Image Not Found")  `onerror` will initiate what it's told to do, so we are telling it to call `imageError` with the `true` flag to change to thumbnail class.  We set to thumbnail because we dont need a massive pagesize error image :)